e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,49 @@
+@ingest
+Feature: Enable Fleet and Deploy Agent
+
+@enroll
+Scenario: Deploying an agent
+  Given the "Fleet" Kibana setup has been executed
+  When an agent is deployed to Fleet
+  Then the agent is listed in Fleet as online
+    And system package dashboards are listed in Fleet
+    And new documents are inserted into Elasticsearch
+
+@start-agent
+Scenario: Starting the agent starts backend processes
+  When an agent is deployed to Fleet
+  Then filebeat is started
+    And metricbeat is started
+
+@stop-agent
+Scenario: Stopping the agent stops backend processes
+  Given an agent is deployed to Fleet
+  When the agent is stopped on the host
+  Then filebeat is stopped
+    And metricbeat is stopped
+
+@unenroll
+Scenario: Un-enrolling an agent
+  Given an agent is deployed to Fleet
+  When the agent is un-enrolled
+  Then the agent is not listed as online in Fleet
```

Comment:
Hey @EricDavisX, could we reuse here the step `the agent is listed in Fleet as "online"`?

Therefore we would have:

```
the agent is listed in Fleet as "online"
the agent is listed in Fleet as "offline"
```
which would be one single step. wdyt?